content/intro-to-storybook/angular/en/screen.md

```diff
@@ -440,9 +440,13 @@ Next, update your `package.json` `scripts` and add a new test task:
 Finally, with your Storybook running, open up a new terminal window and run the following command:
 
 ```bash
-yarn test-storybook --watch
+npm run test-storybook -- --watch
 ```
 
+<div class="aside">
+💡 Interaction testing with the play function is a fantastic way to test your UI components. It can do much more than we've seen here; we recommend reading the <a href="https://storybook.js.org/docs/angular/writing-tests/interaction-testing">official documentation</a> to learn more about it. However, there are more ways you can improve testing. In the <a href="/ui-testing-handbook">Testing Handbook</a>, we'll teach you that, by introducing you to strategies used by scaled-front-end teams that will help supercharge your development workflow.
```

Comment:
```suggestion
💡 Interaction testing with the play function is a fantastic way to test your UI components. It can do much more than we've seen here; we recommend reading the <a href="https://storybook.js.org/docs/angular/writing-tests/interaction-testing">official documentation</a> to learn more about it. 
<br />
For an even deeper dive into testing, check out the <a href="/ui-testing-handbook">Testing Handbook</a>. It covers testing strategies used by scaled-front-end teams to supercharge your development workflow.
```

Reply:
Same for the other frameworks